portfolio/src/main/webapp/script.js

```diff
@@ -48,8 +49,74 @@ function closeModal() {
 // Request content from server
 function getServerData() {
   console.log("Fetching data.");
-  const promise = fetch("/data").then(response => response.json()).then(greeting => {
-    console.log(greeting);
-    document.getElementById('data-container').innerText = greeting;
-  });
-}
\ No newline at end of file
+  const maxElement = document.getElementById("quantity");
+  const maxToDisplay = maxElement.value;
+  const sort = document.getElementById("sort").value;
+  const img = document.getElementById("modal-img").src;
+
+  fetch("/comment?quantity=" + maxToDisplay + "&sort=" + sort + "&img=" + img).then(response => response.json()).then(jsonObj => {
+    const comments = jsonObj.comments;
+    console.log(comments[comments.length - 1]);
+    const total = jsonObj.total;
+    console.log(total);
+
+    const totalElement = document.getElementById("total");
+    totalElement.innerText = ("Comments (" + total + ")");
+
+    // Set maximum number of comments to show as the total number of comments (returned from fetch)
+    maxElement.max = total.toString();
+    maxElement.placeholder = total.toString();
+
+    // Add individual comments to html page
+    const list = document.getElementById("all-comments");
+    list.innerHTML = "";
```

Comment:
can u provide a screenshot of how ur comments section looks list

Its hard for me to imagine how the html works together. After u provide a screenshot ill look at the logic where u generated the html